packages/toolkit/.changes/next-release/Removal-3ef48be0-d3f3-4252-be69-9a0684406944.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Removal",
+	"description": "Cloud9: Remove all support and compatibility."
```

Let's be a bit more coy, since this is potentially confusing
```suggestion
	"description": "Cloud9: remove special-case logic."
```